client/src/modules/journal/trial-balance/trial-balance.service.js

```diff
@@ -113,6 +114,15 @@ function TrialBalanceService(util, $http, Accounts, Constants) {
       });
   }
 
+  /**
+   * @function unpostTransactions
+   *
+   * @description
+   * This function attempts to post to the General Ledger by
```

Comment:
I think this should read "attempts to unpost records from the general ledger to allow modification in the posting journal".